src/collectors/dropwizard/dropwizard.py

```diff
@@ -32,65 +102,53 @@ def get_default_config(self):
         """
         config = super(DropwizardCollector, self).get_default_config()
         config.update({
-            'host':     '127.0.0.1',
-            'port':     8081,
-            'path':     'dropwizard',
+            'host': '127.0.0.1',
+            'port': 8081
         })
         return config
 
     def collect(self):
         if json is None:
             self.log.error('Unable to import json')
             return {}
-        url = 'http://%s:%i/metrics' % (
-            self.config['host'], int(self.config['port']))
+
+        url = 'http://{}:{}/metrics'.format(self.config['host'],
+                                            self.config['port'])
         try:
             response = urllib2.urlopen(url)
         except urllib2.HTTPError as err:
             self.log.error("%s: %s", url, err)
             return
 
         try:
-            result = json.load(response)
+            result_json = json.load(response)
         except (TypeError, ValueError):
             self.log.error("Unable to parse response from elasticsearch as" +
```

Comment:
`elasticsearch` ?? :-) looks like some old copy-paste leftover. would recommend fixing 🗡

Reply:
Haha missed that one, fixed.